introduction.tex

```diff
@@ -84,9 +84,11 @@ \subsection{Register Definition Format}
         \hline
         R/W1C & Read/Write. For each bit in the field, writing 1 clears that bit. Writing 0 has no effect. \\
         \hline
-        W & Write-only. When read this field returns 0. \\
+        WARZ & Write any, read zero. A debugger may write any value. When read
+        this field returns 0. \\
         \hline
-        W1 & Write-only. Only writing 1 has an effect. \\
+        W1 & Write-only. Only writing 1 has an effect. When read the returned
+        value should be 0. \\
```

Comment:
Use the same text as the WARZ case?  "When read this field returns 0"

Reply:
That would not be backwards compatible. Existing implementations might already be returning 1 in that case. I don't think it really matters what they return, but it would be neater if they always return 0.